stream.go

```diff
@@ -231,6 +237,9 @@ func (stream *Stream) connect() (io.ReadCloser, error) {
 		stream.req.Header.Set("Last-Event-ID", stream.lastEventID)
 	}
 	req := *stream.req
+	if stream.queryParamsFunc != nil {
+		req.URL.RawQuery = (*stream.queryParamsFunc)().Encode()
```

Comment:
Should this be a merge? Or do we want replacement behavior?

Reply:
I guess we need replacement to be able to remove a parameter.

Reply:
Updated so that the previously defined query parameters are sent along on subsequent requests.